cli_flags.go

```diff
@@ -63,11 +63,11 @@ var (
 		"If zero, monotonic-realtime clock sync will be performed once, " +
 		"on agent startup, but not periodically."
 	sendErrorFramesHelp = "Send error frames (devfiler only, breaks Kibana)"
-	offCPUThresholdHelp = fmt.Sprintf("If set to a value between 1 and %d will enable "+
-		"off-cpu profiling: Every time an off-cpu entry point is hit, a random number between "+
-		"0 and %d is chosen. If the given threshold is greater than this random number, the "+
-		"off-cpu trace is collected and reported.",
-		support.OffCPUThresholdMax-1, support.OffCPUThresholdMax-1)
+	offCPUThresholdHelp = fmt.Sprintf("A value between 1 and %d enables "+
+		"off-cpu profiling: It is the per-mille chance for an off-cpu event being reported. "+
+		"For example, 1 reports one out of %d off-cpu events. "+
+		"Set to 0 to disable off-cpu profiling.",
+		support.OffCPUThresholdMax, support.OffCPUThresholdMax)
```

Comment:
Reformulate to a more actionable wording, aligning with other help texts, and name default value.
```suggestion
	offCPUThresholdHelp = fmt.Sprintf("Set to a value between 1 and %d to enable "+
		"off-cpu profiling: It is the per-mille chance for an off-cpu event being reported. "+
		"For example, 1 reports one out of %d off-cpu events. "+
		"Default is %d.",
		support.OffCPUThresholdMax, support.OffCPUThresholdMax, defaultOffCPUThreshold)
```

Reply:
Amended, further shortened the help text.